content/ja/docs/concepts/storage/projected-volumes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 投影ボリューム
+content_type: concept
+weight: 21 # just after persistent volumes
+---
+
+<!-- overview -->
+
+このドキュメントでは、Kubernetesの*投影ボリューム*について説明します。[ボリューム](/docs/concepts/storage/volumes/)に精通していることをお勧めします。
+
+<!-- body -->
+
+## 概要
+
+ボリュームは、いくつかの既存の`投影`ボリュームソースを同じディレクトリにマップします。
+
+現在、次のタイプのボリュームソースを投影できます。
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume designドキュメント](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md)を参照してください。
```

Comment:
I have made change on the upstream.
https://github.com/kubernetes/website/pull/31496/files

```suggestion
すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md) デザインドキュメントを参照してください。
```